Sources/Modlog.php

```diff
@@ -58,9 +60,11 @@ function ViewModlog()
 
 		$smcFunc['db_query']('', '
 			DELETE FROM {db_prefix}log_actions
-			WHERE id_log = {int:moderate_log}',
+			WHERE id_log = {int:moderate_log}
+			AND action NOT IN ({array_string:uneditable})'),
```

Comment:
A parse error occurred: Syntax error, unexpected ',' on line 64 at column 50

Reply:
You've got an extra ) there before the comma